server/members/urls.py

```diff
@@ -9,5 +9,5 @@
         name='member-retrieve-update-destroy'
         ),
     path('profile/', views.AuthenticatedMemberProfile.as_view(), name='authenticated-profile'),
-
+    path('verify-discord/', views.UpdateDiscordID.as_view(), name='update-discord-id'),
```

Comment:
not crucial, but I don't like having `-` in endpoints. also, best to keep endpoints "noun oriented"